### DISCUSSION TITLE
GRPC Roadmapped

### DISCUSSION THREAD

**Discussion by wendy-cameron-bzy** [specmatic/specmatic#733] [Ideas and Suggestions]
Heya Given you are working on GRPC how hard is to do tRPC as well?

**Comment by joelrosario** [comment]
Hi @wendy-cameron-bzy tRPC looks pretty interesting. We are looking into it. Are you working on a tRPC-based stack?

**Reply by wendy-cameron-bzy** [reply]
I was at British Airways but given the tech stack here at Beazley wouldnt be suprised if we considered going that way.
